<html> element should not have 'manifest' IDL attribute

rniwa opened the pull request:
#### 332debf04f061ac75ba6130142c68b310d5f0090
<pre>
&lt;html&gt; element should not have &apos;manifest&apos; IDL attribute
<a href="https://bugs.webkit.org/show_bug.cgi?id=58129">https://bugs.webkit.org/show_bug.cgi?id=58129</a>

Reviewed by Chris Dumez.

Removed manifest IDL attribute from the html element.

Blink removed this attribute in 2014:
<a href="https://github.com/chromium/chromium/commit/46d5f250f5b3171c2acfb48d9c97c52d35fd3db7">https://github.com/chromium/chromium/commit/46d5f250f5b3171c2acfb48d9c97c52d35fd3db7</a>

* LayoutTests/fast/dom/HTMLHtmlElement/html-element-manifest-expected.txt: Added.
* LayoutTests/fast/dom/HTMLHtmlElement/html-element-manifest.html: Added.
* LayoutTests/fast/dom/URL-attribute-reflection-expected.txt:
* LayoutTests/fast/dom/URL-attribute-reflection.html:
* Source/WebCore/html/HTMLHtmlElement.idl:

Canonical link: <a href="https://commits.webkit.org/253053@main">https://commits.webkit.org/253053@main</a>
</pre>

cdumez reviewed (approved):
Yes but please add a test to confirm that the attribute does NOT exist.

rniwa committed:
<html> element should not have 'manifest' IDL attribute
https://bugs.webkit.org/show_bug.cgi?id=58129

Reviewed by Chris Dumez.

Removed manifest IDL attribute from the html element.

Blink removed this attribute in 2014:
https://github.com/chromium/chromium/commit/46d5f250f5b3171c2acfb48d9c97c52d35fd3db7

* LayoutTests/fast/dom/HTMLHtmlElement/html-element-manifest-expected.txt: Added.
* LayoutTests/fast/dom/HTMLHtmlElement/html-element-manifest.html: Added.
* LayoutTests/fast/dom/URL-attribute-reflection-expected.txt:
* LayoutTests/fast/dom/URL-attribute-reflection.html:
* Source/WebCore/html/HTMLHtmlElement.idl:

Canonical link: https://commits.webkit.org/253053@main
